app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self.musical_instrument = musical_instrument
+
+    def play_elf_song(self) -> str:

```

Comment:
Invalid type hint for return type. It returns None
```suggestion
    def play_elf_song(self) -> None:

```

Reply:
Still not fixed